lib/spell-check-rules.js

```diff
@@ -12,5 +12,21 @@ module.exports = [
 	{
 		test: /\bjavascript\b/gi,
 		value: 'JavaScript'
+	},
+	{
+		test: /\bmac\s?OS\b/gi,
+		value: 'macOS'
+	},
+	{
+		test: /\bYou\s?Tube\b/gi,
+		value: 'YouTube'
+	},
+	{
+		test: /\bFFmpeg\b/gi,
```

Comment:
Let's remove `FFmpeg` since it's entirely possible to be talking about the `ffmpeg` CLI, in which case this would introduce false positives.

Reply:
@transitive-bullshit Couldn't you wrap it in an inline code block then like you've done above?

Reply:
@transitive-bullshit I've added @sindresorhus's suggestion to the tests to verify it works [here](https://github.com/sindresorhus/awesome-lint/pull/40/commits/1a0d3615339818dd55fd70056f448622b3a2cbaf#diff-d214edc8c496f8c9bdef2cb9b64a14ecR35).

Reply:
This is certainly an interesting idea, I'm just trying to optimize for never having any false positives, and as someone who's published 12+ FFmpeg-related modules, I think this might be too strict.

The difficulty with ensuring lowercase `ffmpeg` is wrapped in a code block would be that the current spell-check rule doesn't differentiate between these two cases. This could definitely be solved by adding some more logic, I'm just on the fence about whether or not to do so at this time.

Since it would complicate things, maybe we remove it from this PR and follow-up in #36?

Reply:
Done. I'll add some follow-up notes on #36.